resolver/ng/init.sls

```diff
@@ -1,5 +1,10 @@
 {% from slspath + "/map.jinja" import resolver with context %}
 
+{% set resolvconf_group = 'root' %}
+{% if grains['os_family'] in ('FreeBSD') %}
+  {% set resolvconf_group = 'root' %}
```

Comment:
This still root at the end right?

Reply:
Ah drats, I forgot to change that to `wheel`.

I'll push up a change and incorporate the map suggestions above.